### DISCUSSION TITLE
Break WooCommerce widgets into smaller components - checkout, cart And my account

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21283] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team!

Current widgets for checkout, cart and my account are too big. For example, the checkout widget has only one widget with a whole bunch of customization styles that we really get lost in. I think you released the checkout, cart and my account widgets in a hurry. Because it has many bugs and most importantly, as I said, several widgets are included in one widget.

### Describe the solution you'd like

Split the current widget into multiple widgets. For example, the checkout widget should be split into 5 new widgets.

1- billing forn
2- shipping form
3- Additional Information
4- order
6- Coupon

If you decide to do this, I highly recommend checking out the jet woobuilder plugin. This plugin is unique in creating WooCommerce pages. Therefore, it can be a suitable pattern for Elementor.

**This also applies to cart and my account widgets.** I hope users will welcome this request and it will be added to Elementor's roadmap.

**Comment by befuddled-and-clueless** [comment]
Yes to above, PLUS add multistep form option for checkout
